README.md

```diff
@@ -39,17 +39,17 @@ Export the component from `app/main.ts`:
 export * from './components/hi-mom.svelte';
 ```
 
-Now use it in anywhere in your HTML or Markdown. 
+Now use it anywhere in your HTML or Markdown. 
 
 ```html
 <hi-mom greeting="i made a web component"></hi-mom>
 ```
 
-**Note:** A weird caveat with Svelte web components is that all styles must be encapsulated. You can use Tailwind, BUT only with `@apply` in the component. Global styles will not work.
+**Note:** A weird caveat with Svelte web components is that all styles must be encapsulated. You can use Tailwind only with `@apply` in the component. Global styles will not work.
 
 ## Commands
 
-- `npm start`: Main dev server. Runs everything you need. 
+- `npm start`: Main dev server. It runs everything you need. 
 - `npm run dev`: Runs components in isolation. Serves `app/index.html` as a playground for components. 
-- `npm run hugo`: Only runs static site. 
+- `npm run Hugo: Only runs the static site. 
```

Comment:
Two things here (Line 52-54):
- `npm run Hugo` likely won't work as `hugo` is listed in `package.json` as `hugo`, without capitalisation.
- These are dot points, I doubt the extra verbosity is required (but that's just an opinion).